### PR TITLE
Contrib brl fix warnings

### PR DESCRIPTION
**Pull request by clucock1**
Corrected most warnings in contrib brl.
Changed tabs to spaces in contrib brl.

### CODE REVIEW

File: contrib/brl/bseg/bstm_multi/bstm_multi_tree_util.cxx [hunk 1]
```diff
@@ -34,7 +36,7 @@ bool ste_from_string(const std::string &s, space_time_enum &ste) {
 std::vector<space_time_enum> parse_subdivisions(const std::string &s) {
   std::vector<space_time_enum> result;
   std::stringstream ss(s);
-  while (ss.good() > 0) {
+  while ((ss.good() ? 1 : 0) > 0) {
```
Comment (decrispell) [inline contrib/brl/bseg/bstm_multi/bstm_multi_tree_util.cxx]: Why not
```
while (ss.good()) {
```
?

Comment (clucock1) [reply contrib/brl/bseg/bstm_multi/bstm_multi_tree_util.cxx]: I wasn't sure if I should delete them but I will.

Comment (clucock1) [reply contrib/brl/bseg/bstm_multi/bstm_multi_tree_util.cxx]: That would be better.

File: contrib/brl/bseg/boxm2/pro/processes/boxm2_load_scene_process.cxx [hunk 2]
```diff
@@ -47,6 +47,7 @@ bool boxm2_load_scene_process(bprb_func_process& pro)
     scene = new boxm2_scene(scene_file);
   }
   catch(const std::ifstream::failure &e) {
+	  e.what();
```
Comment (decrispell) [inline contrib/brl/bseg/boxm2/pro/processes/boxm2_load_scene_process.cxx]: what does this accomplish?  (Did you mean to print it out?)

Comment (clucock1) [reply contrib/brl/bseg/boxm2/pro/processes/boxm2_load_scene_process.cxx]: I think it corrected a warning. I will go back and look at it again.

Comment (clucock1) [reply contrib/brl/bseg/boxm2/pro/processes/boxm2_load_scene_process.cxx]: It corrects the following warning:
1>C:\Users\cynthia.lucock\projects\vxl_fix_warnings\vxl\contrib\brl\bseg\boxm2\pro\processes\boxm2_load_scene_process.cxx(49): warning C4101: 'e': unreferenced local variable

**Pull request merged**